src/core_plugins/kibana/public/management/sections/settings/advanced_row.html

```diff
@@ -69,12 +69,12 @@
         ng-model="conf.unsavedValue"
         ng-options="option as option for option in conf.options"
         class="form-control"
-        data-test-subj="selectInput">
+        data-test-subj="advancedSetting {{conf.name}} selectInput">
```

Comment:
I had to make this kind of change in this file to get the functional tests to pass.